### PR TITLE
CLOSE-QUARTERS COMBAT

### PR DESCRIPTION
**Pull request by Militaires**
now we're cooking with gas

Adds a third martial art, CQC, comes with 5 moves.

`
+	usr << "<span class='notice'>Slam</span>: Grab Harm. Slam opponent into the ground, weakens and knocks down."
+	usr << "<span class='notice'>CQC Kick</span>: Harm Disarm Harm. Knocks opponent away. Knocks out weakened opponents."
+	usr << "<span class='notice'>Restrain</span>: Grab Grab Grab. Locks opponents into a restraining position, disarm to knock them out with a choke hold."
+	usr << "<span class='notice'>Pressure</span>: Disarm Grab. Strike the back of the opponent's neck with your arm. Decent stamina damage."
+	usr << "<span class='notice'>Consecutive CQC</span>: Harm Harm Disarm. Mainly offensive move, high damage and decent stamina damage."
`
Also has a hoist of other abilities such as immediate aggressive grabs (also found in plasma fist and carp), disarming someone doesnt make them drop the item on the floor, you grab it from them and it ends up in your hands and etc.

I plan to give this to nuke ops at a higher price (or lower??) than carp sometime, note that it does not deflect projectiles though so I'm not sure what price to put it at (you can still shoot a gun tho soo)

also I plan to add some more features such as slitting someone's throat after restraining them maybe, when I can figure out stuff like that - that is.

### CODE REVIEW

**Comment by ma44** [comment]
>higher price than carp sometime,
Not just buying a L6 saw because nuke ops rarely go into melee unless they have an energy sword

**Comment by Militaires** [comment]
@ma44 

Still need feedback on price, it can be equal or less. Do know that this is pretty stronk in terms of damage.

It also works for stealth ops bretty good.

**Comment by PKPenguin321** [comment]
Seems redundant

**Comment by Iamgoofball** [comment]
REMEMBER THE BASICS

**Comment by Militaires** [comment]
@PKPenguin321 

image

image

image

@Iamgoofball 

OF CQC

**Comment by TechnoAlchemisto** [comment]
The deflection is what makes carp worth paying the BIG DOLLAR.

Honestly you might as well give it to nukies for free considering how little they actually get into close combat.

**Comment by Hatfish** [comment]
Don't we already have Krav Maga?

**Comment by Militaires** [comment]
@TechnoAlchemisto 
true

@Hatfish 
krav memega is a meme

**Comment by Cheridan** [comment]
@Militaires stealth ops is awful gameplay though and we shouldn't promote it

**Comment by Militaires** [comment]
@Cheridan 

s t e a l t h o p s

**Comment by PKPenguin321** [comment]
I intend to make nuke ops gloves that they spawn with give them krav maga

**Comment by Militaires** [comment]
@PKPenguin321 

wew

**Comment by Militaires** [comment]
how many tcs give this for tater

**Comment by Iamgoofball** [comment]
Can I add wrasslin to ops yet

**Comment by ma44** [comment]
Free; no one is robust enough

**Comment by Xhuis** [comment]
Glad this isn't replacing Sleeping Carp, I liked the flavor for that. I think it's kind of wasteful to have another style (assuming that we actually have Krav Mag and I'm not bonkers) but other than that I'm neutral to this.

File: code/datums/martial.dm [hunk 1]
```diff
@@ -436,6 +436,26 @@
 		name = "empty scroll"
 		icon_state = "blankscroll"
 
+/obj/item/weapon/CQC_scroll
```
Comment (Xhuis) [inline code/datums/martial.dm]: This is just mundane CQC, so maybe make this less magical. Maybe a training manual or something?

**Comment by PKPenguin321** [comment]
We already have krav this is just a meme basically

File: code/datums/martial.dm (hunk 1 above)
Comment (Militaires) [inline code/datums/martial.dm]: @Xhuis 
True, I'll have it be a manual or something yeah.

**Comment by Militaires** [comment]
@PKPenguin321 
krav only has 3 moves doe.

and there's wrasslin and plasma fist but people dont use them alot.

this using combos makes it more diverse as you have multiple ways to go through a situation (kick someone into someone else for a 2-man takedown, then knock the kicked person unconcious, rush the other one with a slam as soon as they get up then get a choke hold etc etc)

**Comment by TehZombehz** [comment]
Only one person made a Metal Gear reference? Step it up.

**Comment by Militaires** [comment]
okay @Xhuis 

dis better?
it's a manual with a self-destruct after reading it now
cqcmanual

**Comment by Xhuis** [comment]
Sure, that'll do.

**Comment by Militaires** [comment]
fucking conflicts hold on

**Comment by Militaires** [comment]
the merge conflict is resolved, how do I mark a .dmi conflict as resolved

**Comment by Militaires** [comment]
i swear to god i resolved it manually but fucking git jesus christ.

when it comes to .dm you can tortoisegit >mark resolved but what do fucking dmis

**Comment by Militaires** [comment]
fuck it new branch

**Comment by optimumtact** [comment]
Martial arts are rarely used, so I don't see we can justify adding just another alternate form for nuke ops.

if you do consider remaking this pr, simply replace the one the nukeops have.

**Pull request closed without merging**